src/sql/parts/modelComponents/inputbox.component.ts

```diff
@@ -13,7 +13,8 @@ import * as sqlops from 'sqlops';
 import { ComponentBase } from 'sql/parts/modelComponents/componentBase';
 import { IComponent, IComponentDescriptor, IModelStore, ComponentEventType } from 'sql/parts/modelComponents/interfaces';
 
-import { InputBox, IInputOptions, MessageType } from 'vs/base/browser/ui/inputbox/inputBox';
+import { InputBox } from 'sql/base/browser/ui/inputbox/inputBox';
+import { IInputOptions, MessageType } from 'vs/base/browser/ui/inputbox/inputBox';
```

Comment:
revert this

Reply:
why? I wanted to use the sql input box

---
In reply to: [191550709](https://github.com/Microsoft/sqlopsstudio/pull/1508#discussion_r191550709) [](ancestors = 191550709)

Reply:
oh I didn't see it's from sql

---
In reply to: [191554363](https://github.com/Microsoft/sqlopsstudio/pull/1508#discussion_r191554363) [](ancestors = 191554363,191550709)